app/react/src/client/preview/render.tsx

```diff
@@ -1,9 +1,7 @@
 import { document } from 'global';
-import React from 'react';
+import React, { ErrorInfo } from 'react';
```

Comment:
Is this used (`ErrorInfo`)? What version of `react` does it mean we require?

Reply:
not really, forgot to remove (and didn't get caught by eslint because of https://github.com/storybookjs/storybook/pull/6621/files#diff-e4403a877d80de653400d88d85e4801aR118, cc @ndelangen)